Stefan/src/main/java/JavaBasics.java

```diff
@@ -0,0 +1,83 @@
+public class JavaBasics {
+
```

Comment:
remove redundant empty line across the class

Reply:
Done